src/gobstuf/stuf/brp/response_mapping.py

```diff
@@ -838,6 +842,9 @@ def answer_code(self) -> str:
     def related(self) -> dict:
         return {}
 
+    def get_links(self, mapped_object: dict):
```

Comment:
Should we return an empty dict here? On line 538 this method is called and on line 541 we're using the variable as a dict without any checks. Seems safer to return an empty dict (as it's very possibly used in more places like this).

Reply:
yes that's better, fixed